### PR TITLE
Align the Inheritance margin's header

### PR DESCRIPTION
**Pull request by Cosifne**
After:
image

Before:
image

Note: the height of the header is a little bit larger than before.

### CODE REVIEW

File: src/VisualStudio/Core/Def/Implementation/InheritanceMargin/MarginGlyph/InheritanceMargin.xaml [hunk 1]
```diff
@@ -115,7 +115,7 @@
                                     </Grid.ColumnDefinitions>
 
                                     <Rectangle Name="Bg" Stroke="Transparent" Fill="Transparent" StrokeThickness="0" Grid.ColumnSpan="5"/>
-                                    <ContentPresenter Grid.Column="0" x:Name="Icon" Margin="22,1,4,1" Width="16" Height="16" VerticalAlignment="Center"
+                                    <ContentPresenter Grid.Column="0" x:Name="Icon" Margin="28,1,4,1" Width="16" Height="16" VerticalAlignment="Center"
```
Comment (Cosifne) [inline src/VisualStudio/Core/Def/Implementation/InheritanceMargin/MarginGlyph/InheritanceMargin.xaml:118]: The left value is calculated 
4 (header icon left margin) + 16 (width of header icon) + 4 (header icon right margin) + 4 (Header text block left margin)
= 28

**Comment by CyrusNajmabadi** [comment]
I prefer the way it looks before. Things were invented a natural amount (like in a tree view). The new way looks far too indented to me. Do we need to change this!

**Comment by ryzngard** [comment]
Agree with @CyrusNajmabadi , I don't think this looks better. It looks like before the right edge of the title icon aligned with the left edge of the item icon. 

```
[ i ]  ImplementedInterfaces
    ^------------------------------- Right edge
     [ m ] I1
     ^------------------------------- Left edge
```

now we have 

```
[ i ]  ImplementedInterfaces
       ^------------------------------- Right edge
       [ m ] I1
       ^------------------------------- Left edge
```

It leaves a _lot_ of empty space on the left that seems unnecessary. Was there feedback on this, or just personal preference?

File: src/VisualStudio/Core/Def/Implementation/InheritanceMargin/MarginGlyph/InheritanceMargin.xaml (hunk 1 above)
Comment (ryzngard) [reply src/VisualStudio/Core/Def/Implementation/InheritanceMargin/MarginGlyph/InheritanceMargin.xaml:118]: Can we try with just 24. Header icon width + margins. That may help address some of the problems with "22" here but avoid adding too much space

**Comment by Cosifne** [comment]
> Agree with @CyrusNajmabadi , I don't think this looks better. It looks like before the right edge of the title icon aligned with the left edge of the item icon.
> 
> ```
> [ i ]  ImplementedInterfaces
>     ^------------------------------- Right edge
>      [ m ] I1
>      ^------------------------------- Left edge
> ```
> 
> now we have
> 
> ```
> [ i ]  ImplementedInterfaces
>        ^------------------------------- Right edge
>        [ m ] I1
>        ^------------------------------- Left edge
> ```
> 
> It leaves a _lot_ of empty space on the left that seems unnecessary. Was there feedback on this, or just personal preference?
@CyrusNajmabadi 
28 might be too large, I have changed the value to 24 based on @ryzngard 's comment.

File: src/VisualStudio/Core/Def/Implementation/InheritanceMargin/MarginGlyph/InheritanceMargin.xaml [hunk 2]
```diff
@@ -220,12 +220,13 @@
                     <Setter Property="AutomationProperties.Name" Value="{Binding AutomationName}"/>
                     <Setter Property="IsEnabled" Value="False"/>
                     <Setter Property="Background" Value="{DynamicResource {x:Static vsui:EnvironmentColors.TitleBarInactiveBrushKey}}"/>
+                    <Setter Property="MinHeight" Value="22"/>
```
Comment (Cosifne) [inline src/VisualStudio/Core/Def/Implementation/InheritanceMargin/MarginGlyph/InheritanceMargin.xaml:223]: Found this value is missed (we have three MenuItem template here, two of them has MinHeight = 22)
And it is causing all the height of Header is 18, and the height of other menuItem is 22.
After:
image

Before:
image

Note: there is a small edge between the icon and the text before. Make all the min height to 22 will resolve the problem

**Review by ryzngard — APPROVED** [review]

**Pull request merged**